style.css

```diff
@@ -0,0 +1,419 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.contents {
+  background-color: #f9f9fa;
+  width: 320px;
+}
+
+.header-file {
+  display: flex;
+  flex-direction: row;
+  height: 80px;
+  background-repeat: no-repeat;
+  background-image: url(IMAGES/image-1.svg);
```

Comment:
> * [ ]  Please add the correct image path to make the background image in the top left corner appear because now it doesn't show up in the online version.
>   [Hint] this is the correct path in this case:
> 
> ```css
> background-image: url(./images/image-1.svg);
> ```
> 
> ![image](https://user-images.githubusercontent.com/68796232/185016191-bcb6818f-730e-4cc9-ad04-99c1106adf0f.png)

the background image file is on the code, it probably didn't load at your end.
background-image: url(./images/image-1.svg);